.github/workflows/package_build.yml

```diff
@@ -77,14 +67,15 @@ jobs:
         path: skare3_tools
     - uses: sot/setup-miniconda@v2
       with:
-        miniconda-version: "py38_4.8.3"
-        python-version: "3.8"
-        channels: https://ska:${{ secrets.CONDA_PASSWORD }}@cxc.cfa.harvard.edu/mta/ASPECT/ska3-conda/flight,conda-forge
-    - name: Build Dependencies
-      run: conda install setuptools_scm gitpython conda-build jinja2 pyyaml python=3.8 numpy packaging testr ska_helpers
+        miniconda-version: py39_4.12.0
+        environment-file: build-environment.yml
+        channels: https://ska:${{ secrets.CONDA_PASSWORD }}@cxc.cfa.harvard.edu/mta/ASPECT/ska3-conda/prime,conda-forge
```

Comment:
I will update that to point to flight after we have promoted. In this case it is easy enough. If I was using the docker workflow, it would be more involved.

One thing to note here is that this is a chicken-and-egg problem when one is creating a channel from scratch, but not an issue in this PR.